game/src/MusicConductor/MusicConductor.gd

```diff
@@ -0,0 +1,64 @@
+extends Node
+# SS-67
+var selectedTrack = 0
+var availableSongs : Array[SongInfo] = []
+var autoPlayNextSong : bool = true
```

Comment:
```suggestion
extends Node

# SS-67
var selectedTrack = 0
var availableSongs : Array[SongInfo] = []
var autoPlayNextSong : bool = true
```

Its cleaner to keep at least a newline between the first line.

Reply:
Addressed in most recent commit